### PR TITLE
Issue #3277704: Remove PHPUNIT_75 constant in Rector

### PR DESCRIPTION
**Pull request by FlorentTorregrosa**
## Description
Remove usage of non-existing constant in rector 0.12.22

## Drupal.org issue
https://www.drupal.org/project/rector/issues/3277704

### CODE REVIEW

File: config/drupal-8/drupal-8-all-deprecations.php [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_60);
     $containerConfigurator->import(PHPUnitSetList::PHPUNIT_70);
-    $containerConfigurator->import(PHPUnitSetList::PHPUNIT_75);
```
Comment (mglaman) [inline config/drupal-8/drupal-8-all-deprecations.php:14]: What is the replacement? We need this as it fixes PHPUnit code for folks.

Comment (mglaman) [inline config/drupal-8/drupal-8-all-deprecations.php:14]: What is the replacement? We need this as it fixes PHPUnit code for folks.

**Review by mglaman — CHANGES_REQUESTED** [review]

**Comment by mglaman** [comment]
We can work on this once https://github.com/palantirnet/drupal-rector/pull/195 lands and is green. This was removed in a patch release without deprecation.

File: config/drupal-8/drupal-8-all-deprecations.php (hunk 1 above)
Comment (mglaman) [reply config/drupal-8/drupal-8-all-deprecations.php:14]: Answering for myself:

There is no replacement value. Removed in rector/rector-phpunit:0.11.26 due to it being a complex upgrade path.

**Comment by mglaman** [comment]
If this was rebased against `main`, it should pass. I cannot do that, however.

**Comment by FlorentTorregrosa** [comment]
Hi,

Thanks @mglaman for your review.

PR rebased

**Review by mglaman — APPROVED** [review]

**Comment by FlorentTorregrosa** [comment]
Thanks for the merge :)

**Pull request merged**